openassessment/xblock/static/js/src/oa_server.js

```diff
@@ -216,6 +217,28 @@ export class Server {
     }).promise();
   }
 
+  /**
+   * Reset a student's submission.
+   *
+    * @returns {promise} A promise which resolves with no arguments if successful,
+   *     and which fails with an error message otherwise.
+   */
+  resetSubmission() {
+    const url = this.url('reset_submission');
+    return $.Deferred((defer) => {
+      $.ajax({
+        type: 'POST',
+        url,
+        data: JSON.stringify({}),
+        contentType: jsonContentType,
```

Comment:
` jsonContentType` looks undefined here

Reply:
`jsonContentType` is a global const in the file